EvotoApi/Areas/Management/Controllers/ManaVotesController.cs

```diff
@@ -1,24 +1,34 @@
 ﻿using System;
-using System.Configuration;
-using System.Diagnostics;
 using System.Linq;
-using System.Net;
 using System.Threading.Tasks;
 using System.Web.Http;
-using Common;
 using Common.Exceptions;
 using EvotoApi.Areas.Management.Connections;
-using EvotoApi.Areas.ManagementApi.Models;
 using EvotoApi.Areas.ManagementApi.Models.Request;
 using EvotoApi.Areas.ManagementApi.Models.Response;
 using Management.Database.Interfaces;
 using Management.Models;
-using Newtonsoft.Json;
-using RestSharp;
+using System.Web.Http.Controllers;
+using System.Threading;
+using Microsoft.AspNet.Identity;
 
 namespace EvotoApi.Areas.ManagementApi.Controllers
 {
+    public class Authorize2 : AuthorizeAttribute
```

Comment:
I didn't mean to commit this, and looking at the commit history, I'm not sure how this got here?
Needs removing and switching back to Authrorize in the class attribute (was just doing this for debug breakpoints)